tests/integration_tests/functional/test_api.py

```diff
@@ -1193,6 +1193,7 @@ def test_get_full_config_after_restoring_snapshot(microvm_factory, uvm_nano):
     expected_cfg["boot-source"] = {
         "kernel_image_path": uvm_nano.get_jailed_resource(uvm_nano.kernel_file),
         "initrd_path": None,
+        "boot_args": None,
```

Comment:
ah, this answers my first question :)

I guess this is fine because we are adding a new field to an existing response, which can be done in a minor